src/test/kotlin/uk/gov/justice/digital/hmpps/hmppsaccreditedprogrammesapi/pact/PactContractTest.kt

```diff
@@ -29,56 +29,47 @@ class PactContractTest {
-  fun `ensure course participation exists with ID 1c0fbebe-7768-4dbe-ae58-6036183dbeff`() {}
+  @State("Offering 790a2dfe-7de5-4504-bb9c-83e6e53a6537 exists for course d3abc217-75ee-46e9-a010-368f30282367")
+  fun `ensure offering 790a2dfe-7de5-4504-bb9c-83e6e53a6537 exists for course d3abc217-75ee-46e9-a010-368f30282367`() {}
 
-  @State("A referral exists with ID 0c46ed09-170b-4c0f-aee8-a24eeaeeddaa")
-  fun `ensure referral exists with ID 0c46ed09-170b-4c0f-aee8-a24eeaeeddaa`() {}
+  @State("Offering 790a2dfe-7de5-4504-bb9c-83e6e53a6537 exists")
+  fun `ensure offering 790a2dfe-7de5-4504-bb9c-83e6e53a6537 exists`() {}
 
-  @State("An offering exists with ID 7fffcc6a-11f8-4713-be35-cf5ff1aee517")
-  fun `ensure an offering exists with ID 7fffcc6a-11f8-4713-be35-cf5ff1aee517`() {}
+  @State("Participation 0cff5da9-1e90-4ee2-a5cb-94dc49c4b004 exists")
+  fun `ensure participation 0cff5da9-1e90-4ee2-a5cb-94dc49c4b004 exists`() {}
 
-  @State("An offering with ID 7fffcc6a-11f8-4713-be35-cf5ff1aee517 exists and has an associated course")
-  fun `ensure An offering with ID 7fffcc6a-11f8-4713-be35-cf5ff1aee517 exists and has an associated course`() {}
+  @State("Participation 0cff5da9-1e90-4ee2-a5cb-94dc49c4b004 exists")
+  fun `ensure participation 0cff5da9-1e90-4ee2-a5cb-94dc49c4b004 exists`() {}
 
-  @State("Offerings exist for a course with ID d3abc217-75ee-46e9-a010-368f30282367")
-  fun `ensure offerings exist for a course with ID d3abc217-75ee-46e9-a010-368f30282367`() {}
+  @State("Person A1234AA has participations 0cff5da9-1e90-4ee2-a5cb-94dc49c4b004 and eb357e5d-5416-43bf-a8d2-0dc8fd92162e and no others")
+  fun `ensure person A1234AA has participations 0cff5da9-1e90-4ee2-a5cb-94dc49c4b004 and eb357e5d-5416-43bf-a8d2-0dc8fd92162e and no others`() {}
 
-  @State("Referral can be created")
-  fun `ensure referral can be created`() {}
+  @State("Referral 0c46ed09-170b-4c0f-aee8-a24eeaeeddaa exists with status referral_started")
+  fun `ensure referral 0c46ed09-170b-4c0f-aee8-a24eeaeeddaa exists with status referral_started`() {}
 
-  @State("Referral can be updated")
-  fun `ensure referral can be updated`() {}
+  @State("Referral 0c46ed09-170b-4c0f-aee8-a24eeaeeddaa exists")
+  fun `ensure referral 0c46ed09-170b-4c0f-aee8-a24eeaeeddaa exists`() {}
 
-  @State("Participations exist for a person with prison number A1234AA")
-  fun `ensure participations exist for a person with prison number A1234AA`() {}
-
-  @State("Referral status can be updated")
-  fun `ensure referral status can be updated`() {}
-
-  @State("Referral can be submitted")
-  fun `ensure referral can be submitted`() {}
-
-  @State("Referral summaries exist for an organisation with the ID a026cc07-8e0b-40dd-9b66-1b3dacecc63d")
-  fun `ensure referral summaries exist for an organisation with the ID a026cc07-8e0b-40dd-9b66-1b3dacecc63d`() {}
+  @State("Referral summaries exist for organisation a026cc07-8e0b-40dd-9b66-1b3dacecc63d")
```

Comment:
As mentioned [on the UI PR](https://github.com/ministryofjustice/hmpps-accredited-programmes-ui/pull/377#discussion_r1409012388) let's make this organisation ID `BWN` to match what's in the test data on an offering.